.unreleased/pr_7701

```diff
@@ -0,0 +1 @@
+Implements: #7701 Implement bool compression
```

Comment:
Let's expand on this a little, something like "Implement a custom compression algorithm for bool columns. It is experimental and can undergo backwards-incompatible changes. For testing, enable it using `timescaledb.enable_bool_compression = on`."